Original patch by James Henstridge to fix a SEGV bug in libXft.
Backported to 6.8.2 by Mike A. Harris <mharris@redhat.com>

Fixes bugs:
	https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=155634
	https://bugs.freedesktop.org/show_bug.cgi?id=1951
	https://bugs.freedesktop.org/show_bug.cgi?id=1958



--- xc/lib/Xft/xftfreetype.c.xft-releasefile-crash	2004-08-11 19:37:34.000000000 -0400
+++ xc/lib/Xft/xftfreetype.c	2005-04-25 11:04:28.000000000 -0400
@@ -290,7 +290,7 @@
 	    FT_Done_Face (f->face);
     }
     XftMemFree (XFT_MEM_FILE,
-		sizeof (XftFtFile) + f->file ? strlen (f->file) + 1 : 0);
+		sizeof (XftFtFile) + (f->file ? strlen (f->file) + 1 : 0));
     free (f);
 }
 
